docs/detections/rules-ui-create.asciidoc

```diff
@@ -536,9 +536,10 @@ the rule. For example, links to background information.
 .. *False positive examples* (optional): List of common scenarios that may produce
 false-positive alerts.
 .. *MITRE ATT&CK^TM^ threats* (optional): Add relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.
-.. *Custom highlighted fields* (optional): Specify highlighted fields for personalized alert investigation flows. Fields with values are added to the <<investigation-section,Highlighted fields>> section within the alert details flyout. Fields without values aren't added. After you create the rule, you can find all custom highlighted fields in the About section of the rule details page.
+.. *Custom highlighted fields* (optional): Specify one or more highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the indices you selected for the rule's data source.  
```

Comment:
Optional Nit: "Personalized" sounds kinda like it could be tailored to an individual user, but custom highlighted fields are probably system-wide. "Customized" seems more accurate but basically repeats "custom" -- maybe "unique"?

Reply:
Unique works 👍🏼 

```suggestion
.. *Custom highlighted fields* (optional): Specify one or more highlighted fields for unique alert investigation flows. You can choose any fields that are available in the indices you selected for the rule's data source.  
```